config/provisioners/natss/100-natss.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: nats-streaming
+  name: nats-streaming
+  namespace: knative-eventing
+data:
+  gnatsd.conf: |
+    # configuration file used to override default NATS server settings
+  stan.conf: |
+    # content of configuration file used to override default NATS Streaming server settings
+---  
+apiVersion: v1
+kind: Service
+metadata:
+  name: nats-streaming
+  namespace: knative-eventing
```

Comment:
not sure if the `nats-streaming` bit should be in the knative-eventing ns

Reply:
NATSS is running now in its own namespace: natss
Done